dashboard/app/assets/javascripts/dashboard.controller.js

```diff
@@ -0,0 +1,49 @@
+(function() {
+  'use strict';
+
+  angular.module('qat-dashboard').controller('DashboardController', [
+    'testExecutor',
+    function(testExecutor) {
+      var vm = this;
```

Comment:
@christianvuerings This would results in failing of instanceof tests: http://stackoverflow.com/questions/2938940/return-statement-in-js-constructors

Would that be ok?